lib/utils.js

```diff
@@ -0,0 +1,32 @@
+let ftl = require('../lib/ftl');
+// General functions that are command agnostic.
+module.exports.stripContent = (messageContent) => {
+  const stringParts = messageContent.split(' ');
+  stringParts.shift();
+
+  const userPost = stringParts.join(' ');
+  return userPost;
+};
+
+module.exports.memberName = (msg) => {
+  if (msg.channel.type === 0) {
+    return msg.member.nick ? msg.member.nick : msg.author.username; //Prefers member nickname when available
```

Comment:
```suggestion
    return msg.member.nick || msg.author.username; //Prefers member nickname when available
```

This can be a bit less repetitive. You may also be able to use the `??` operator instead of `||` which more closely matches the semantics we want, but I don't know if it is available in the version of Node that Timelord is using.